app/facades/new_authorization_request/api_entreprise_facade.rb

```diff
@@ -3,13 +3,13 @@ class APIEntrepriseFacade < Base
     ALREADY_INTEGRATED_EDITORS = %w[achatpublic actradis arnia atline_services aws_achat axyus dematis entr_ouvert klekoon maximilien megalis_bretagne pictav_informatique smartglobal solution_attestations].freeze
 
     def already_integrated_editors
-      @already_integrated_editors ||= Editor.all.select do |editor|
+      @already_integrated_editors ||= ServiceProvider.all.select do |editor|
```

Comment:
Ici j'introduirais une méthode `editors` sur `ServiceProvider` qui fait ceci:

```ruby
def self.editors
  self.all.where(type: 'editor')
end
```

Et du coup ici:

```suggestion
      @already_integrated_editors ||= ServiceProvider.editors.select do |editor|
```

histoire d'être plus accurate